clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -1446,6 +1446,51 @@ TEST(TransferTest, BaseClassInitializer) {
       llvm::Succeeded());
 }
 
+TEST(TransferTest, StructModeledFieldsWithAccessor) {
+  std::string Code = R"(
+    class S {
+      int *P;
+      int *Q;
+      int X;
+      int Y;
+      int Z;
```

Comment:
Rename these to match the methods that return them (e.g. `Ptr`, `PtrNonConst` etc.)? This would make it easier to read the assertion below.